src/main/java/uk/gov/hmcts/reform/sscs/controller/EvidenceUploadController.java

```diff
@@ -33,6 +39,77 @@ public EvidenceUploadController(
         this.coversheetService = coversheetService;
     }
 
+    @ApiOperation(value = "Upload evidence",
+            notes = "Uploads evidence for an appeal which will be held in a draft state against the case that is not "
+                    + "visible by a caseworker in CCD. You will need to submit the evidence for it to be visbale in CCD "
+                    + "by a caseworker. You need to have an appeal in CCD."
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Evidence has been added to the appeal"),
+            @ApiResponse(code = 404, message = "No online hearing found with online hearing id"),
+            @ApiResponse(code = 422, message = "The file cannot be added to the document store")
+    })
+    @PutMapping(
+            value = "{identifier}/evidence",
+            consumes = MediaType.MULTIPART_FORM_DATA_VALUE,
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity<Evidence> uploadEvidence(
+            @ApiParam(value = "either the online hearing or CCD case id", example = "xxxxx-xxxx-xxxx-xxxx") @PathVariable("identifier") String identifier,
+            @RequestParam("file") MultipartFile file
+    ) {
+        return uploadEvidence(() -> evidenceUploadService.uploadDraftHearingEvidence(identifier, file));
```

Comment:
Why is this called 'uploadDraftHearingEvidence'? This isn't anything to do with a Hearing right?

Reply:
No, it just uploading the draft document from MYA